src/vs/workbench/api/common/extHostNotebookEditor.ts

```diff
@@ -94,7 +97,7 @@ export class ExtHostNotebookEditor {
 	}
 
 	_acceptSelections(selections: vscode.NotebookRange[]): void {
-		this._selections = selections.length === 0 ? [new NotebookRange(0, 0)] : selections;
+		this._selections = this._validSelections(selections);
```

Comment:
You might wanna handle this one different because it would mean the renderer/us sends a broken selection and that's arguably a programming/logic flaw on our end. For extension-provided values it's a different story